onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -6651,18 +6651,15 @@ def aten_positive(self: TensorType) -> TensorType:
 
 
 @torch_op(
-    ("aten::pow.Tensor_Tensor", "aten::pow.Tensor_Scalar"),
+    ("aten::pow.Tensor_Tensor", "aten::pow.Tensor_Scalar", "_operator::pow"),
```

Comment:
Should we also update the annotation if exponent may not be a tensor?

Reply:
Sure

Reply:
https://github.com/pytorch/pytorch/blob/af1072ffb6ecbc5c98f9de48a50797112dfe0544/aten/src/ATen/native/native_functions.yaml#L10256

Reply:
Done. PTAL